src/map/pc.h

```diff
@@ -833,8 +839,8 @@ struct pc_interface {
 	/* */
 
 BEGIN_ZEROED_BLOCK; /* Everything within this block will be memset to 0 when status_defaults() is executed */
-	uint64 exp_table[CLASS_COUNT][2][MAX_LEVEL];
-	int max_level[CLASS_COUNT][2];
+	VECTOR_DECL(struct class_exp_group) class_exp_groups[2];
+	const struct class_exp_group *class_exp_table[CLASS_COUNT][2];
```

Comment:
also this array is big in size, better move it outside of pc interface into pointer to structs with arrays.
example in skill.h ``struct s_skill_dbs *dbs;``
this need because HPM copy whole interface in memory, this mean it using 2 times of interface size. because this better put into interface directly pointers or functions. all other in connected structs to interface